chrome/locale/fr/options.dtd

```diff
@@ -45,14 +45,14 @@
 
 <!ENTITY display.label "Affichage">
 
-<!ENTITY showDKIMHeader.label  "Montrer l'entête DKIM">
-<!ENTITY showDKIMStatusbarpanel.label  "Montrer la signature DKIM dans la barre de status">
-<!ENTITY showDKIMFromTooltip.label  "Montrer l'infobulle DKIM dans l'expéditeur de l'entête">
-<!ENTITY showDKIM.value.0.label  "jamais">
-<!ENTITY showDKIM.value.10.label "quand un courriel avec une signature DKIM valide est vu">
-<!ENTITY showDKIM.value.30.label "quand un courriel avec une signature DKIM est vu">
-<!ENTITY showDKIM.value.40.label "quand le courriel est vu">
-<!ENTITY showDKIM.value.50.label "quand le message est vu">
+<!ENTITY showDKIMHeader.label  "Montrer l'entête DKIM:">
+<!ENTITY showDKIMStatusbarpanel.label  "Montrer la signature DKIM dans la barre de status:">
+<!ENTITY showDKIMFromTooltip.label  "Montrer l'infobulle DKIM dans l'expéditeur de l'entête:">
+<!ENTITY showDKIM.value.0.label  "Jamais">
+<!ENTITY showDKIM.value.10.label "Quand un courriel avec une signature DKIM valide est vu">
+<!ENTITY showDKIM.value.30.label "Quand un courriel avec une signature DKIM est vu">
+<!ENTITY showDKIM.value.40.label "Quand le courriel est vu">
+<!ENTITY showDKIM.value.50.label "Quand le message est vu">
```

Comment:
Unless you know the langue an it's capitalization rules I think we should leave it as is. Same for hu.

For it and uk I would like to wait for how the maintainers of the translations handles it when I'm asking to update it for 5.4.0

Reply:
I removed all capitalizations, but left the colons - I hope, you're fine with that. I can also leave the other languages completely untouched.

Reply:
Keeping the colons is fine. This is also what I did on master, as all languages there using colons in the same places as the en local.